src/ch05/15.py

```diff
@@ -0,0 +1,36 @@
+import pylab
+import numpy as np
+from random import *
+pi=np.pi
+n=1000
+ts=np.linspace(0,12*pi,n)
+xs=np.sin(ts)*(np.exp(np.cos(ts))-2*np.cos(4*ts)-np.sin(ts/12)**5)
+ys=np.cos(ts)*(np.exp(np.cos(ts))-2*np.cos(4*ts)-np.sin(ts/12)**5)
+
+
+map=[[0x8,0x8,0x7f,0x49,0x7f,0x8,0x8],[0x8,0x8,0x3e,0x2a,0x7f,0x14,0x63]]
+R,C=7,7
+ds=5
+#m=int(input("> "))
+m = 1
+ds=20
+pylab.axis("off")

```

Comment:
Suggest adding `pylab.axis("equal")` for isometric scaling.

```suggestion
pylab.axis("off")
pylab.axis("equal")
```